docs/source/sysadmin_guide/pbft_adding_removing_node.rst

```diff
@@ -0,0 +1,158 @@
+
+.. _adding-a-pbft-node-label:
+
+Adding a PBFT Node
+==================
+
+To add a new node to an existing PBFT network, you will install and configure
+the node, start it and wait for it to catch up with the rest of the network,
+then update ``sawtooth.consensus.pbft.members`` on an existing node.
+
+You can add several nodes at the same time.
+
+1. Install and configure Sawtooth on the new node, as described in
+   :doc:`setting_up_sawtooth_poet-sim`.
+   Skip the procedure to create a genesis block.
+
+#. Start the new node. It will join the Sawtooth network and start the block
+   catch-up procedure, where it receives and commits the blocks that are already
+   on the blockchain. (For details, see the `PBFT node catch-up RFC
+   <https://github.com/hyperledger/sawtooth-rfcs/blob/master/text/0031-pbft-node-catchup.md>`_.)
+
+#. Wait for the node to catch up to the rest of the network (get within a few
+   blocks of the chain head). If the new node becomes a PBFT member before it
+   has caught up, the entire network could slow down or stall while the new
+   member node catches up.
+
+   This process can take a while, depending on the blockchain size, node's
+   hardware, and network speed. You can look at the logs or run
+   :ref:`sawtooth block list <sawtooth-block-list-label>` to check on the
+   new node's progress.
+
+#. Copy the new node's public validator key for use in the next step.
+
+   .. code-block:: console
+
+      $ cat /etc/sawtooth/keys/validator.pub
+
+   This step assumes that the validator key is stored in the default location,
+   ``/etc/sawtooth/keys``. If not, use the location specified by the ``key_dir``
+   setting (see :doc:`configuring_sawtooth/path_configuration_file`).
+
+#. On an existing member node, update ``sawtooth.consensus.pbft.members`` to
+   include the public validator key of the new node.
+
+   Run the following steps on a node that has permission to change on-chain
+   settings; that is, a node whose validator key is listed in
+   ``sawtooth.identity.allowed_keys``. Usually, the node that created the
+   genesis block is on this list. For more information, see
+   :ref:`config-onchain-txn-perm-label`.
+
+   a. List the current PBFT member nodes:
+
+      .. code-block:: console
+
+         $ sawtooth setting list --filter sawtooth.consensus.pbft.members
+
+   #. Submit a transaction that specifies the new list of all PBFT member nodes
+      (the previous list plus the new node's key).
+
+      .. Important::
+
+         BE VERY CAREFUL! Make sure to specify the full list of keys.
+         Double-check each key before you run this command, because a typo could
+         stall the network.
+
+      .. code-block:: console
+
+         $ sawset proposal create \
+           --key /etc/sawtooth/keys/validator.priv \
+           sawtooth.consensus.pbft.members=[previous-list, NEW-KEY]
+
+      If there are no errors, the block containing this setting change will be
+      committed quickly.
```

Comment:
We can't really make a guarantee that it will be committed quickly. I'd just say something like: "If there are no errors, this settings change will be committed to the chain."